.github/workflows/build-and-test.yml

```diff
@@ -157,10 +157,6 @@ jobs:
 
       - name: Install from source distribution
         run : |
-          # Prevent pip from thinking that CWD is a natcap.invest
-          # installation. It's not.
-          rm -r natcap.invest.egg-info

```

Comment:
This doesn't seem to be a problem any more, maybe because `natcap.invest.egg-info` is now in `src/`.

Reply:
Neat!  I wonder when they made that change.